core/flyout_base.js

```diff
@@ -441,6 +441,16 @@ Blockly.Flyout.prototype.positionAt_ = function(width, height, x, y) {
     // Set the scrollbars origin to be the top left of the flyout.
     scrollbar.setOrigin(x, y);
     scrollbar.resize();
+    // If origin changed and metrics haven't changed enough to trigger
+    // reposition in resize, we need to call setPosition. See issue #4692.
+    if (scrollbar.hScroll) {
+      scrollbar.hScroll.setPosition(
+          scrollbar.hScroll.position.x, scrollbar.hScroll.position.y);
+    } else if (scrollbar.vScroll) {
+      scrollbar.vScroll.setPosition(
+          scrollbar.vScroll.position.x, scrollbar.vScroll.position.y);
+
+    }
```

Comment:
What happens if you have both horizontal and vertical scrollbars? Does it not have to reposition?

Reply:
That's a good point. I assumed there would only be single direction scrollbars in flyouts, but I can easily change this to support flyouts with paired scrollbars too.